mlir/lib/Target/CppOutput/gridwise_convolution_implicit_gemm_v4r4_gen_xdlops.cpp

```diff
@@ -664,20 +665,18 @@ std::unique_ptr<llvm::StringRef> mlir::translateModuleToMIOpenHeaderXDLOPS(Modul
       auto filterLayoutAttr = op.getAttrOfType<ArrayAttr>("filter_layout");
       auto inputLayoutAttr = op.getAttrOfType<ArrayAttr>("input_layout");
 
-      size_t dimKF, dimCF, dimYF, dimXF;
-      size_t dimNI, dimCI, dimHI, dimWI;
-
+      size_t dimKF, dimNI, dimCI;
       for (size_t i = 0; i < 4; ++i) {
         auto filterDim = filterLayoutAttr.getValue()[i].dyn_cast<StringAttr>().getValue();
 
         if (filterDim.str() == "k") {
           dimKF = i;
         } else if (filterDim.str() == "c") {
-          dimCF = i;
+          // dimCF = i;
```

Comment:
@jerryyin these couple of lines looks shady to me... care to explain the rationales behind them?

Reply:
To fix `clang-tidy` issues. Those commented out variables are never used elsewhere and `clang-tidy` is not happy about it. So I did a quick hack...

Reply:
how about comment out unused codes, and use a comment to say this C++ output logic only does NCHW so some cases won't happen at all.

Reply:
Sure, I'd delete those and add a comment above.